File: api/DTO/LoginResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace api.DTO
{
    public class LoginResponse
    {
        public String TokenType { get; set; } = String.Empty;
        public String AccessToken { get; set; } = String.Empty;
        public long ExpiresInMinutes { get; set; }
    }
}

File: app/lib/api-response/login_response.dart
class LoginResponse {
  String? tokenType;
  String? accessToken;
  int? expiresInMinutes;

  LoginResponse({
    this.tokenType,
    this.accessToken,
    this.expiresInMinutes,
  });

  LoginResponse.fromJson(Map<String, dynamic> json) {
    tokenType = json['tokenType'] ?? '';
    accessToken = json['accessToken'] ?? '';
    expiresInMinutes = json['expiresInMinutes'] ?? 0;
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['tokenType'] = tokenType;
    data['accessToken'] = accessToken;
    data['expiresInMinutes'] = expiresInMinutes;
    return data;
  }
}